webknossos-tracingstore/app/com/scalableminds/webknossos/tracingstore/tracings/skeleton/updating/SkeletonUpdateActions.scala

```diff
@@ -217,7 +221,8 @@ case class CreateNodeSkeletonAction(id: Int,
                                     actionTimestamp: Option[Long] = None,
                                     actionAuthorId: Option[String] = None,
                                     info: Option[String] = None,
-                                    additionalCoordinates: Option[Seq[AdditionalCoordinate]] = None)
+                                    additionalCoordinates: Option[Seq[AdditionalCoordinate]] = None,
+                                    metadata: Option[Seq[MetadataEntry]] = None)
```

Comment:
See my previous comments This isn't supported by the current version of the frontend